README.md

```diff
@@ -29,8 +29,47 @@ The following packages are used internally and are not meant to be used directly
 
 ## Usage
 
+### Basic setup
+
 This example uses React, but the tools are platform agnostic. You should already know the basics of Redux before using this library!
 
+In order to integrate `redux-tools` into your project, you first need to add necessary dependencies:
+
+```
+yarn add @redux-tools/reducers @redux-tools/reducers-react @redux-tools/actions
```

Comment:
actions are not technically necessary for this example

Reply:
That is true. 
It is used in following example, but it's not needed for the basics.

Reply:
you can probably add a comment explaining that they are just utility functions for actions and reducers, highly recommended

Reply:
i wouldn't go more in depth until the "proper" docs are finished :)